index.html

```diff
@@ -42,6 +42,26 @@
                 border-left-width: 4px;
                 border-left-style: solid;
             }
+            #UNCONFIRMED {
+                border-style: solid;
+                border-width: 1;
+	        }	
+            #CONFIRMED {
+                border-style: solid;
+                border-width: 1;
+	        }	
+            #IN_PROGRESS {
+                border-style: solid;
+                border-width: 1;
+	        }	
+            #RESOLVED {
+                border-style: solid;
+                border-width: 1;
+	        }	
+            #VERIFIED {
+                border-style: solid;
+                border-width: 1;
+	        }	
```

Comment:
These aren't for the cards, they're for the div's of the board columns. That way when someone drags they know where each column is. Right now when scrolled down, and the headers are out of view, hard to tell where the correct target column starts/begins.